checker/watcher.go

```diff
@@ -237,3 +246,12 @@ func (pw *PeersWatcher) Shutdown() error {
 
 	return nil
 }
+
+func shouldConsider(mdInfo *mdInfo) bool {
+	consider := false
```

Comment:
nit: looks like there is no real need for consider variable. You can just simply `return mdInfo.ipsecState == "active" && mdInfo.connCheckState == "active"`

Reply:
Yes, I agree :) Initially I had other states as well to consider. Will clean up.